Error in launch file using <ros_gz_bridge>

<!-- If you're not sure whether your problem is a bug, please ask a question at
https://robotics.stackexchange.com instead.-->

## Environment
* OS Version: <!-- e.g, Ubuntu 20.04 --> Ubuntu 24.04
* Source or binary build? Source
  <!-- If binary, which version? -->
  <!-- If source, which branch and what is the latest commit? -->

## Description

We might have broken something since #628 . If I run this launch file (saved as `ros_gz_bridge.launch`:

```xml
<launch>
  <ros_gz_bridge
    bridge_name="ros_gz_bridge"
    config_file="/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_bridge/test/config/full.yaml"/>
</launch>
```

with:
```bash
ros2 launch ros_gz_bridge ros_gz_bridge.launch
```

I'm getting this error message:
```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): value='False' is not an instance of <class 'bool'>
```

@Amronos, could you try this on your end to confirm please?

--- a/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py
+++ b/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py
@@ -175,6 +175,9 @@ class RosGzBridge(Action):
         if isinstance(self.__create_own_container, list):
             self.__create_own_container = self.__create_own_container[0]
 
+        if isinstance(self.__use_respawn, list):
+            self.__use_respawn = self.__use_respawn[0]
+
         # Standard node configuration
         load_nodes = GroupAction(
             condition=IfCondition(PythonExpression(['not ', self.__use_composition])),
@@ -185,7 +188,7 @@ class RosGzBridge(Action):
                     name=self.__bridge_name,
                     namespace=self.__namespace,
                     output='screen',
-                    respawn=self.__use_respawn,
+                    respawn=bool(self.__use_respawn),
                     respawn_delay=2.0,
                     parameters=[{'config_file': self.__config_file, **parsed_bridge_params}],
                     arguments=['--ros-args', '--log-level', self.__log_level],

